cmake/OpenCVExtraTargets.cmake

```diff
@@ -6,7 +6,7 @@ CONFIGURE_FILE(
   "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
   @ONLY)
 
-ADD_CUSTOM_TARGET(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
+ADD_CUSTOM_TARGET(cmake_uninstall "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
```

Comment:
`make uninstall` should work for OpenCV. This change breaks this behaviour.
Probably you use OpenCV via some strange way in your projects. Recommended way is to use `find_package(OpenCV)` CMake statement.
ExternalProject/add_subdirectory are not supported.